<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogLbxEditor</class>
 <widget class="QDialog" name="DialogLbxEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView_LbxImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>255</red>
      <green>192</green>
      <blue>255</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_LbxIndex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>211</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>(index in LBX file)</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton_Load">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>185</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Load LBX</string>
   </property>
   <property name="icon">
    <iconset resource="../Resources/InteractiveGameTweaker.qrc">
     <normaloff>:/heroes/valana.gif</normaloff>:/heroes/valana.gif</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Replace">
   <property name="geometry">
    <rect>
     <x>264</x>
     <y>530</y>
     <width>191</width>
     <height>40</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>200</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Replace in LBX</string>
   </property>
   <property name="icon">
    <iconset resource="../Resources/InteractiveGameTweaker.qrc">
     <normaloff>:/heroes/greyfar.gif</normaloff>:/heroes/greyfar.gif</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_BitmapImage">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>441</width>
     <height>221</height>
    </rect>
   </property>
   <property name="backgroundBrush">
    <brush brushstyle="SolidPattern">
     <color alpha="255">
      <red>255</red>
      <green>192</green>
      <blue>255</blue>
     </color>
    </brush>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_FileIndex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>441</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>(images in directory containing LBX file)</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources>
  <include location="../Resources/InteractiveGameTweaker.qrc"/>
 </resources>
 <connections/>
</ui>
